capstone/src/main/java/com/google/sps/data/User.java

```diff
@@ -46,6 +46,14 @@ public EmbeddedEntity getCachedInterestVector() {
         return (EmbeddedEntity) this.delegate.getProperty(User.TAGS_PROPERTY_KEY);
     }
 
+    public void setMagnitude(Double magnitude) {
```

Comment:
getMagnitude should probably have a name which associates it with the cached interest vector so it doesnt seem like it is a normalized property of user. (at least include the name cached, and probably a name which associates it with the vector as well)